portfolio/src/main/webapp/index.html

```diff
@@ -25,6 +25,12 @@ <h2>✨ Fatima's Fun Facts </h2>
         <p>Click here to read a random fun fact about me:</p>
         <button onclick="addRandomFact()">🌻 Who am I?</button>
         <div id="random-fact-container"></div>
+
+        <h2>Greetings</h2>
+        <p>Click here to see a greeting:</p>
+        <button onclick="getGreetingUsingAsyncAwait() ">Fetch a greeting!</button>
```

Comment:
nit: remove trailing space
```suggestion
        <button onclick="getGreetingUsingAsyncAwait()">Fetch a greeting!</button>
```